analyzers/tests/SonarAnalyzer.UnitTest/TestCases/Hotspots/PubliclyWritableDirectories.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using System.IO;
+
+namespace Tests.Diagnostics
+{
+    public class Program
+    {
+        public void Compliant()
+        {
+            var randomPath = Path.Combine(Path.GetTempPath(), Path.GetRandomFileName()); // Compliant
```

Comment:
Let's also document `Path.GetTempFileName()`

Reply:
Well, is `Path.GetTempFileName()` compliant or sensitive? It directly creates the file for you, so it doesn't have the autodelete component in it.
Unless you open it like that afterwards.

Reply:
I would say it is compliant as this rule targets directories, and [S5445](https://jira.sonarsource.com/browse/RSPEC-5445) targets insecure temporary files.